WalletWasabi/Extensions/NBitcoinExtensions.cs

```diff
@@ -522,6 +534,6 @@ public static byte[] ExtractKeyId(this Script scriptPubKey)
 			}
 		}
 
-		return null;
+		throw new NotImplementedException($"Unsupported script type.");
```

Comment:
It should be `NotSupportedException` but I can see this is not the only place this exception is used so, it's okay, we have to change it everywhere.